src/scenes/app/stores/index.js

```diff
@@ -1,38 +1,65 @@
-import { observable, action } from 'mobx';
-import sampleData from '../../../helpers/dataSourceSample';
+import { observable, action, toJS } from 'mobx';
 import strings from '../../../localization/en/strings';
+import Utils from '../../../helpers/Utils';
 
 class ObservableTodoStore {
-  @observable items = sampleData;
+  @observable items = [];
   @observable error = '';
```

Comment:
Maybe it would be better to define `error` as `null`. Its meaning is more clear that way.